pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -54,121 +54,10 @@ module FakeDashboard
     {user_id: FACILITATOR[:id], permission: 'facilitator'}
   ]
 
-  #
-  # Fake Data: Courses
-  #
-  COURSES = [
-    COURSE_CSP = {
-      id: 15,
-      name: 'csp-2017',
-      created_at: '2016-01-01 00:01:02',
-      updated_at: '2016-01-01 00:01:02'
-    }
-  ]
-
-  #
-  # Fake Data: Scripts
-  #
-  UNITS = [
-    UNIT_FOO = {
-      id: 1,
-      name: 'Foo',
-      published_state: 'preview'
-    },
-    UNIT_BAR = {
-      id: 3,
-      name: 'Bar',
-      published_state: 'preview'
-    },
-    UNIT_MC = {
-      id: 4,
-      name: 'mc',
-      published_state: 'preview'
-    },
-    UNIT_HOUROFCODE = {
-      id: 5,
-      name: 'hourofcode',
-      published_state: 'preview'
-    },
-    UNIT_MINECRAFT = {
-      id: 6,
-      name: 'minecraft',
-      published_state: 'preview'
-    },
-    UNIT_FLAPPY = {
-      id: 10,
-      name: 'flappy',
-      published_state: 'preview'
-    },
-    UNIT_CSP1 = {
-      id: 31,
-      name: 'csp1-2017',
-      published_state: 'preview'
-    },
-    UNIT_CSP2 = {
-      id: 32,
-      name: 'csp2-2017',
-      published_state: 'preview'
-    },
-    UNIT_CSP3 = {
-      id: 34,
-      name: 'csp3-2017',
-      published_state: 'preview'
-    },
-    # put the unlaunched units at the end and give them higher ids, to make
-    # unit testing slightly easier.
-    UNIT_ALLTHETHINGS = {
-      id: 45,
-      name: 'allthehiddenthings',
-      published_state: 'beta'
-    },
-    UNIT_CSP2_ALT = {
-      id: 53,
-      name: 'csp2-alt',
-      published_state: 'beta'
-    },
-  ]
-
-  COURSE_UNITS = [
-    {
-      course_id: COURSE_CSP[:id],
-      script_id: UNIT_CSP1[:id],
-      position: 1
-    },
-    {
-      course_id: COURSE_CSP[:id],
-      script_id: UNIT_CSP2[:id],
-      position: 2
-    },
-    {
-      course_id: COURSE_CSP[:id],
-      script_id: UNIT_CSP2_ALT[:id],
-      position: 2,
-      experiment_name: 'csp2-alt-experiment',
-      default_script_id: UNIT_CSP2[:id]
```

Comment:
the pegasus test changes are a bit of bonus cleanup. I noticed them because we were using alternate units here, then discovered there was a bigger chunk of test code here that is unused.